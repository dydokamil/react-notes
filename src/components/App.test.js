import React from 'react'
import UnconnectedApp, { App } from './App'
import { shallow } from 'enzyme'

describe('App', () => {
  const app = shallow(<App />)
  it('should match the snapshot', () => {
    expect(app).toMatchSnapshot()
  })
  it('should render the connected Left component', () => {
    expect(app.find('Connect(Left)').exists()).toBe(true)
  })
  it('should render the connected Right component', () => {
    expect(app.find('Connect(Right)').exists()).toBe(true)
  })
})
