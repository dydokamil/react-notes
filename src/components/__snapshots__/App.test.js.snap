// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <main>
        <Connect(Left)
                className="left"
        />
        <Connect(Right)
                className="right"
                onUpdateNote={[Function]}
        />
</main>,
      "className": "wrap",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(Left)
            className="left"
/>,
          <Connect(Right)
            className="right"
            onUpdateNote={[Function]}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "className": "left",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "className": "right",
            "onUpdateNote": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "main",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <main>
          <Connect(Left)
                    className="left"
          />
          <Connect(Right)
                    className="right"
                    onUpdateNote={[Function]}
          />
</main>,
        "className": "wrap",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Connect(Left)
              className="left"
/>,
            <Connect(Right)
              className="right"
              onUpdateNote={[Function]}
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "left",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "right",
              "onUpdateNote": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "main",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
